appendix/templates/login.html

```diff
@@ -0,0 +1,10 @@
+{% extends "templates/login.html" %}
+{% block site %}
+
+<div id="ipython-main-app" class="container">
+    <h1>You don't have access to this Binder!</h1>
+    <h2>
+        You can get a new Binder for this repo by clicking <a href="{{binder_url}}">here</a>
```

Comment:
Prefix with: "If you followed a link here that someone shared with you, you can get a new ..."? I'm wondering if someone who just followed a link here (and doesn't know that is the wrong link -> a newcomer) will understand that what they want is a new binder. How do we phrase it so that someone who is new, now confused because they see an "error page" follows the link instead of shrugging and closing the tab.